tardis/tests/tests_slow/test_w7.py

```diff
@@ -49,7 +39,8 @@ def setup(self, request, reference, data_path, atomic_data_fname,
         assert self.atom_data.uuid1 == kurucz_data_file_uuid1
 
         # Create a Configuration through yaml file and atom data.
-        tardis_config = Configuration.from_config_dict(config_yaml, self.atom_data)
+        tardis_config = Configuration.from_config_dict(
```

Comment:
Can you please post the error? From my understanding it should be possible to use Configuration.from_yaml and that uses the relative paths automatically. I even patched `.from_yaml` to allow atom_data to be passed along